packages/contracts/test/manager/DiamondTest.t.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./DiamondTestSetup.sol";
+
+contract TestDiamond is DiamondSetup {
```

Comment:
Tests should also ensure that the correct events are being emitted when necessary and that they're failing when expected to (ie whevere you've written a `require()` statement)

Reply:
> Tests should also ensure that the correct events are being emitted when necessary and that they're failing when expected to (ie whevere you've written a `require()` statement)

Please look at ManagerFacet.sol, there is no require() statements, only getter and setter functions

Reply:
`function diamontCut()` should include a check to ensure signature collision doesn't occur.

Reply:
> `function diamontCut()` should include a check to ensure signature collision doesn't occur.

Originally I added full test cases for diamond with test facets, but when adding managerFacet, I removed all of them because I thought the diamond-3 is absolutely true